### PR TITLE
Activate TMPresentationReconcilerTypeScriptTest

### PR DESCRIPTION
**Pull request by pascalleclercq**
Activate TMPresentationReconcilerTypeScriptTest 

- mark  org.eclipse.tm4e.ui.tests as "eclipse-test-plugin"
- rename TMPresentationReconcilerTypeScriptTests to TMPresentationReconcilerTypeScriptTest to conform to maven's defaults

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,5 +3,10 @@ language: java
 jdk:
   - oraclejdk8
 
+
```
Comment (angelozerr) [inline .travis.yml]: We should discuss about that with @mickaelistria

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-
-You can install textmate.java with the update site http://oss.opensagres.fr/textmate/1.0.0-SNAPSHOT/
-which provides samples of syntax coloration with:
-
- * GenericEditor for Language Server (C#, CSS, JSON)
- * a TypeScript Editor.
- 
+# TextMate support in Eclipse IDE (tm4e)
+https://travis-ci.org/pascalleclercq/tm4e
+[![Build Status](https://travis-ci.org/pascalleclercq/tm4e.png)](https://travis-ci.org/pascalleclercq/tm4e)
```
Comment (angelozerr) [inline README.md]: URL with pascalleclercq is bad. We should wait for @mickaelistria work about how to build tm4e inside eclipse

**Comment by pascalleclercq** [comment]
Hi, 
sorry I wasn't aware my commits named "Configure TravisCI" was included in the pull request.

I only wanted to activate Travis CI on my own fork.

At first I wanted to create 2 different pull requests :
- "Activate TMPresentationReconcilerTypeScriptTest"
- "Migrate to Eclipse Public License - v 1.0"

Obviously I need to learn more about how pull requests works on github :(

**Comment by angelozerr** [comment]
>Obviously I need to learn more about how pull requests works on github :(

No problem @pascalleclercq please let me change README.md

**Pull request closed without merging**